### DISCUSSION TITLE
site站点无法显示例子

### DISCUSSION THREAD

**Discussion by zmcode** [antvis/G6#5183] [Q&A]
![image](https://github.com/antvis/G6/assets/48686959/b5ef1188-1cb2-4b38-bbda-3ad274a776f3)
根据文档拉项目下来跑本地站点, 但是无法显示example中的例子
![image](https://github.com/antvis/G6/assets/48686959/2552456b-d462-4dc9-aec1-923621128141)

**Comment by zmcode** [comment]
想贡献代码, 该如何实时编译并生成例子呢?
例如我想写一个插件, 实时编译plugin的代码, 可以到那个目录下 npm run watch, 但是最终输出的文件是g6目录下的dist, 我要每次都build?

**Reply by Aarebecca** [reply]
现在你看的是 v4 的分支哈，这个分支下你可以在 plugin 包下执行 `npm run watch`，当 pacakges/plugin 下的代码有变更时会自动 build 并更新 es 和 lib 内的产物。

接着启动 site，在站点里的目录应该可以实时预览，子包内容更新时站点会同时更新的

**Reply by Aarebecca** [reply]
v5 分支下面的话一方插件位于 `g6/src/stdlib/plugin` 下，因此可以采取跟 v4 相同的方式
也可以基于集成测试环境，使用 `npm run dev` 启动，具体的写法可以参考 g6/tests/integration 里的测试用例

**Comment by zmcode** [comment]
有没有官方群可以拉我一下?

**Comment by Aarebecca** [comment]
你好，可以加一下这个群哈
![image](https://github.com/antvis/G6/assets/25787943/dbece223-9055-4b96-b1f5-01ebfabd0cd5)

**Comment by Aarebecca** [comment]
官网站点已经更新，请再次安装尝试
